Classes/Widgets/Provider/QueueSizeDataProvider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FriendsOfCrawler\CrawlerWidgets\Widgets\Provider;
```

Comment:
repository is crawler-widget, so would keep the namespace singular too.

Reply:
I changed it to FriendsOfCrawler\CrawlerWidget